### PR TITLE
Bugfix/admin docs endpoint

### LINKED ISSUE #43: Fix Wiremock admin docs endpoint
as of according to wiremock docs, it is a first link mentioned... so it could be quite surprising that it is not working in aem-stubs

### PR DESCRIPTION
**Pull request by marcinkp**
Fix #43

### CODE REVIEW

File: wiremock/src/main/java/com/cognifide/aem/stubs/wiremock/admin/DocRequestFilter.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    return mappings.entrySet()
+      .stream()
+      .filter(e -> request.getUrl().endsWith(e.getKey()))
+      .findFirst()
+      .map(Entry::getValue)
+      .map(s -> RequestFilterAction.stopWith(response(s)))
+      .orElseGet(() -> RequestFilterAction.continueWith(request));
+  }
+
+  private ResponseDefinition response(DocFile file) {
```
Comment (pun-ky) [inline wiremock/src/main/java/com/cognifide/aem/stubs/wiremock/admin/DocRequestFilter.java:37]: heh you are stubbing itself ?:D funny

File: wiremock/src/main/java/com/cognifide/aem/stubs/wiremock/admin/DocRequestFilter.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+        .withStatus(200)
+        .withBody(content)
+        .withHeader(CONTENT_TYPE, file.getMime())
+        .build();
+    } catch (IOException e) {
+      return responseDefinition().withStatus(500).build();
+    }
+  }
+
+  private byte[] readFile(String path) throws IOException {
```
Comment (pun-ky) [inline wiremock/src/main/java/com/cognifide/aem/stubs/wiremock/admin/DocRequestFilter.java:50]: why allocating here / byte\[\]?  withBody does not accept streams? / inputstream?

**Review by pun-ky — APPROVED** [review]

**Review by pun-ky — APPROVED** [review]

**Review by pun-ky — APPROVED** [review]
nice

File: wiremock/src/main/java/com/cognifide/aem/stubs/wiremock/admin/DocRequestFilter.java (hunk 1 above)
Comment (marcinkp) [reply wiremock/src/main/java/com/cognifide/aem/stubs/wiremock/admin/DocRequestFilter.java:37]: the same is doing WM. Not my idea :)

**Pull request merged**